### DISCUSSION TITLE
pytest_fixture_setup hook doesn't run for session scoped fixtures

### DISCUSSION THREAD

**Discussion by voronind** [pytest-dev/pytest#7993] [Q&A]
Is it OK that pytest_fixture_setup hook doesn't run for session scoped fixtures?
I added pytest_fixture_setup function to conftest.py and it runs for function and module scoped fixtures but not session-scoped.

**Comment by RonnyPfannschmidt** [comment]
it should be called, more context is needed to determine if its user error or pytest bug

**Reply by voronind** [reply]
#8000
